src/gt4py/next/program_processors/runners/dace_fieldview/transformations/auto_opt.py

```diff
@@ -32,19 +43,18 @@ def gt_simplify(
     Instead of calling `sdfg.simplify()` directly, you should use this function,
     as it is specially tuned for GridTool based SDFGs.
 
+    By default this function will run the normal DaCe simplify pass. However, if
+    `skip` is not set or `None` then the parts listed in `GT_SIMPLIFY_DEFAULT_SKIP_SET`
+    will be skipped.
+
     Args:
         sdfg: The SDFG to optimize.
         validate: Perform validation after the pass has run.
         validate_all: Perform extensive validation.
         skip: List of simplify passes that should not be applied.
-
-    Note:
-        The reason for this function is that we can influence how simplify works.
-        Since some parts in simplify might break things in the SDFG.
-        However, currently nothing is customized yet, and the function just calls
-        the simplification pass directly.
     """
-
+    if skip is None:
+        skip = GT_SIMPLIFY_DEFAULT_SKIP_SET
```

Comment:
We could use `GT_SIMPLIFY_DEFAULT_SKIP_SET` as default instead of `None` on line 39, or is it style-thing?

Reply:
Actually we could do that.